### PR TITLE
Make sure we normalize the locales from 'default_locale' for lookups.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This should fix mozilla/addons#170 and should also fix mozilla/addons#177.

### CODE REVIEW

File: src/olympia/files/utils.py [hunk 1]
```diff
@@ -968,8 +968,9 @@ def resolve_i18n_message(message, messages, locale, default_locale=None):
     if match is None:
         return message
 
+    locale = find_language(locale)
+    default_locale = find_language(locale)
```
Comment (diox) [inline src/olympia/files/utils.py]: Shouldn't it be `find_language(default_locale)` ? Also, if `default_locale` is `None`, that'd be bad - I assume it's checked by the caller ?

Comment (EnTeQuAk) [inline src/olympia/files/utils.py]: Fixed in a39c2b3

File: src/olympia/files/tests/test_utils_.py [hunk 2]
```diff
@@ -528,7 +528,7 @@ def test_resolve_i18n_message_locale_found():
 
 def test_resolve_i18n_message_uses_default_locale():
     messages = {
-        'en': {
+        'en-US': {
```
Comment (EnTeQuAk) [inline src/olympia/files/tests/test_utils_.py]: This is how `messages` is being normalized already and passed in, the caller correctly assumes this.

**Comment by diox** [comment]
r+ (usual intermittent test failure)

**Pull request merged**